src/content/desci/index.md

```diff
@@ -8,77 +8,268 @@ sidebar: true
+
+Decentralized science can help to address these issues by allowing for more diverse funding sources (from DAOs, quadratic donations to crowdfunding and more), easier access to data and methods, and by providing incentives for reproducibility.
+
+​​
+
+
+## Why we need science
+
+Science is a process that allows us to create and translate knowledge in order to improve the quality of life for humanity as a whole. It also provides solutions to some of the most complex problems we face. In order to be trustworthy, knowledge produced by science must be replicable and open to scrutiny. This process should be impartial and available to the public. Any knowledge that cannot be replicated or tested for falsifiability should not be considered as the truth.
+
+By creating knowledge that is impartial and available to the public, we enable ourselves to make fundamental technological advancements. These advancements give us the power to more effectively navigate and manipulate our environment, leading to vast improvements in quality of life. As we continue to create impartial knowledge and make technological advancements, we find ourselves capable of managing increasingly complex systems. Today, we are confronted with species-level existential risks, such as climate change and global pandemics. Science is the only process we have that offers equitable positive outcomes.
+
+
+## How DeSci improves science
+
+**List of key problems in science and how decentralized science can help to address these issues**
+
+
+
+* bureaucratic funding structures -> allow for more diverse funding sources (quadratic donations, DAOs, crowdfunding etc.) 
+* replication crisis -> allowing for easier access to data and methods, and by providing incentives for reproducibility.
+* misaligned incentives -> aligning incentives more closely with the goals of science. 
+* poor data quality -> incentivising transparent data sharing and quality control. 
+* publication bias -> enabling publication in a wider range of outlets. 
+* p-hacking -> incentivising transparent and honest reporting of results.
+* cherry picking -> requiring publication of all results, including negative or inconclusive ones. 
+* HARKing: hypothesizing after the results are known -> providing incentives for transparent and honest reporting of results.
+
+A comparison to traditional science
+
+
+<table>
+  <tr>
+   <td><strong>DeSci</strong>
+   </td>
+   <td><strong>Traditional science</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>You raise funding from a global community.
+   </td>
+   <td>You raise funding from a national agency.
+   </td>
+  </tr>
+  <tr>
+   <td>You collaborate with peers from all over the globe in dynamic teams.
+   </td>
+   <td>Your collaborations are limited by funding organizations and home institutions.
```

Comment:
Again, I have never experienced a funder or institution restricting my collaborations on the basis of their affiliation, nor can I recall any instances of this in my wider network- is it really a pervasive problem?

Reply:
I agree. DeSci will add more options for a researcher, but it's not that the traditional system leaves or is absolutely restrictive.